src/mono/netcore/System.Private.CoreLib/src/System/Reflection/Emit/ConstructorBuilder.Mono.cs

```diff
@@ -56,9 +56,11 @@ public sealed partial class ConstructorBuilder : ConstructorInfo
         private Type[][]? paramModReq;
         private Type[][]? paramModOpt;
         private object? permissions;
-#pragma warning restore 169, 414
+#endregion
+
         internal bool finished;
 
+        [DynamicDependency(nameof(permissions))] // Automatically keeps all previous fields too due to StructLayout
```

Comment:
Any concerns with the robustness of this, if/when fields are reorganized?

Reply:
Not really. I think there are enough comments to catch that and right now we keep all fields